examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -89,7 +89,15 @@ CommissioningParameters PairingCommand::GetCommissioningParameters()
 
 CHIP_ERROR PairingCommand::PaseWithCode(NodeId remoteId)
 {
-    return CurrentCommissioner().EstablishPASEConnection(remoteId, mOnboardingPayload);
+    chip::SetupPayload payload;
+
+    QRCodeSetupPayloadParser(mOnboardingPayload).populatePayload(payload);
+    mSetupPINCode  = payload.setUpPINCode;
+    mDiscriminator = payload.discriminator;
+
+    RendezvousParameters rendezvousParams =
+        RendezvousParameters().SetSetupPINCode(mSetupPINCode).SetDiscriminator(mDiscriminator).SetPeerAddress(PeerAddress::BLE());
```

Comment:
Won't this break PASE with code to onnetwork things?  Why are the changes here needed?

Reply:
Removed the SetPeerAddress(). Is that enough @bzbarsky-apple ?

Reply:
Did this get tested?  Why ask me if it's enough?